examples/webgl_buffergeometry_points_interleaved.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - buffergeometry [particles]</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				color: #cccccc;
+				font-family:Monospace;
+				font-size:13px;
+				text-align:center;
+
+				background-color: #050505;
+				margin: 0px;
+				overflow: hidden;
+			}
+
+			#info {
+				position: absolute;
+				top: 0px; width: 100%;
+				padding: 5px;
+			}
+
+			a {
+				color: #0080ff;
+			}
+		</style>
+	</head>
+	<body>
+
+		<div id="container"></div>
+		<div id="info"><a href="http://threejs.org" target="_blank">three.js</a> webgl - buffergeometry - particles</div>
+
+		<script src="../build/three.js"></script>
+
+		<script src="js/Detector.js"></script>
+		<script src="js/libs/stats.min.js"></script>
+
+		<script>
+
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var container, stats;
+
+			var camera, scene, renderer;
+
+			var mesh;
+
+			init();
+			animate();
+
+			function init() {
+
+				container = document.getElementById( 'container' );
+
+				camera = new THREE.PerspectiveCamera( 27, window.innerWidth / window.innerHeight, 5, 3500 );
+				camera.position.z = 2750;
+
+				scene = new THREE.Scene();
+				scene.fog = new THREE.Fog( 0x050505, 2000, 3500 );
+
+				//
+
+				var particles = 500000;
+
+				var geometry = new THREE.BufferGeometry();
+
+				var interleaved_buffer_float32 = new Float32Array( particles * 4 );
+				interleaved_buffer_uint8 = new Uint8Array(interleaved_buffer_float32.buffer)
+
+				var color = new THREE.Color();
+
+				var n = 1000, n2 = n / 2; // particles spread in the cube
+
+				for ( var i = 0; i < interleaved_buffer_float32.length; i += 4 ) {
+
+					// positions
+
+					var x = Math.random() * n - n2;
+					var y = Math.random() * n - n2;
+					var z = Math.random() * n - n2;
+
+					interleaved_buffer_float32[ i ]     = x;
+					interleaved_buffer_float32[ i + 1 ] = y;
+					interleaved_buffer_float32[ i + 2 ] = z;
+
+					// colors
+
+					var vx = ( x / n ) + 0.5;
+					var vy = ( y / n ) + 0.5;
+					var vz = ( z / n ) + 0.5;
+
+					color.setRGB( vx, vy, vz );
+
+					interleaved_buffer_uint8[ 4*(i + 3)+0] = color.r*255;
+					interleaved_buffer_uint8[ 4*(i + 3)+1] = color.g*255;
+					interleaved_buffer_uint8[ 4*(i + 3)+2] = color.b*255;
+					interleaved_buffer_uint8[ 4*(i + 3)+3] = 0;
+
+				}
+
+				ibp = new THREE.InterleavedBuffer(interleaved_buffer_float32, 4);
+				ibc = new THREE.InterleavedBuffer(interleaved_buffer_uint8, 16);
+				
+				geometry.addAttribute( 'position', new THREE.InterleavedBufferAttribute(ibp, 3, 0, false ) );
+				geometry.addAttribute( 'color', new THREE.InterleavedBufferAttribute(ibc, 3, 12, true ) );
+
```

Comment:
I though the point was the attributes would share the same buffer -- which I guess they can't because the data is of different types. So, what is being interleaved? Is the intention to modify this example?